apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -13,6 +13,7 @@
 
 PL_ANSWER_CORRECT_DEFAULT = True
 PL_ANSWER_INDENT_DEFAULT = -1
+PL_ALLOW_EMPTY_SUBMISSION_DEFAULT = False
```

Comment:
I'd prefer to have this be `ALLOW_BLANK_DEFAULT = False`
to match what other elements have